dappy_test.go

```diff
@@ -19,6 +19,15 @@ func TestDappyAuth_HappyPath(t *testing.T) {
 		"should authenticate successfully")
 }
 
+func TestDappyUsers(t *testing.T) {
```

Comment:
so, since we have testify going here, let's go ahead and make use of it.
instead of doing the standard library thing here, maybe something more like this:
```
users, err := client.Users() <<< because we are going to switch those return values
assert.Nil(t, err, "there should be no error when fetching users")
assert.GreaterOrEqual(t, len(users), 1, "we should get back at least 1 user") <<< maybe more?
```
might even say we go so far as to hard code a couple users that come back (haven't run it yet, so i don't know off the top of my head, but could do stuff like this too:
```
assert.Contains(t, users, "tesla")
assert.Contains(t, users, "whatever other scientists we get back or whatever")
```
i donno. i like the idea of the tests being fairly brittle, but since these are really integration tests against a pretend server (https://www.forumsys.com/tutorials/integration-how-to/ldap/online-ldap-test-server/) maybe we shouldn't rely too heavily on what they have (though it hasn't changed in years, as that's kind of its point). I donno... thoughts here are appreciated.